packages/twenty-front/src/modules/object-record/record-board/hooks/useSetRecordIdsForColumn.ts

```diff
@@ -0,0 +1,109 @@
+import { useRecoilCallback } from 'recoil';
+
+import { recordGroupDefinitionFamilyState } from '@/object-record/record-group/states/recordGroupDefinitionFamilyState';
+import { recordGroupFieldMetadataComponentState } from '@/object-record/record-group/states/recordGroupFieldMetadataComponentState';
+import { recordGroupIdsComponentState } from '@/object-record/record-group/states/recordGroupIdsComponentState';
+import { recordIndexAllRowIdsComponentState } from '@/object-record/record-index/states/recordIndexAllRowIdsComponentState';
+import { recordIndexRowIdsByGroupComponentFamilyState } from '@/object-record/record-index/states/recordIndexRowIdsByGroupComponentFamilyState';
+import { ObjectRecord } from '@/object-record/types/ObjectRecord';
+import { sortRecordsByPosition } from '@/object-record/utils/sortRecordsByPosition';
+import { useRecoilComponentCallbackStateV2 } from '@/ui/utilities/state/component-state/hooks/useRecoilComponentCallbackStateV2';
+import { getSnapshotValue } from '@/ui/utilities/state/utils/getSnapshotValue';
+import { isDeeplyEqual } from '~/utils/isDeeplyEqual';
+import { isDefined } from '~/utils/isDefined';
+
+export const useSetRecordIdsForColumn = (recordBoardId?: string) => {
```

Comment:
looks similar to useSetRecordBoardRecordIds?

naming is not consistent: useSetRecordIdsForColumn + do we need to do it twice?

Reply:
ok, could you add a TODO?